znoyder/config.yml

```diff
@@ -53,64 +62,77 @@ include:
 exclude:
   'cinder':  # OSPCRE-22
     'osp-17.0':
-      'osp-tox-pep8': 'Storage DFG decided to opt-out for now'
-      'osp-tox-py39': 'Storage DFG decided to opt-out for now'
+      'openstack-tox-pep8': 'Storage DFG decided to opt-out for now'
+      'openstack-tox-py39': 'Storage DFG decided to opt-out for now'
+      'osp-tox-functional-py39': 'Storage DFG decided to opt-out for now'
```

Comment:
All the `openstack-tox-` should be `osp-tox-` here and below

Reply:
Oh, it's true, I saw it in your PR, but I forgot to update, I'll do it now